package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@yext/search-headless-react",
-  "version": "2.1.0",
+  "version": "2.1.1-beta.0",
```

Comment:
I believe this should be 2.2.0-beta.0 because this contains new features. If it were a patch with just bug fixes, then it would be 2.1.1-beta.0

Reply:
Done!